base/reflection.jl

```diff
@@ -1253,6 +1253,51 @@ function hasmethod(@nospecialize(f), @nospecialize(t), kwnames::Tuple{Vararg{Sym
     return issubset(kwnames, kws)
 end
 
+"""
+    fbody = bodyfunction(mnokw::Method)
+
+Find the keyword "body function" (the function that contains the body of the method
+as written, called after all missing keyword-arguments have been assigned default values).
+`mnokw` is the method that gets called when you invoke it without supplying any keywords.
+"""
+function bodyfunction(mnokw::Method)
```

Comment:
`meth_no_kw` maybe.. Right now, it feels like I'm reading some foreign language :P

Reply:
I went with `basemethod` and referenced `which` and `methods`, seem reasonable?